templates/event.php

```diff
@@ -9,39 +9,72 @@
 use WP_User;
 use Wporg\TranslationEvents\Attendee\Attendee;
 use Wporg\TranslationEvents\Event\Event;
-use Wporg\TranslationEvents\Event\Event_End_Date;
-use Wporg\TranslationEvents\Event\Event_Start_Date;
 use Wporg\TranslationEvents\Stats\Event_Stats;
 use Wporg\TranslationEvents\Stats\Stats_Row;
 
 /** @var bool $user_is_attending */
 /** @var bool $user_is_contributor */
+/** @var Attendee[] $hosts */
 /** @var Attendee[] $attendees_not_contributing */
 /** @var Attendee[] $contributors */
 /** @var array $new_contributor_ids */
 /** @var Event $event */
-/** @var int $event_id */
-/** @var string $event_title */
-/** @var string $event_description */
-/** @var Event_Start_Date $event_start */
-/** @var Event_End_Date $event_end */
 /** @var Event_Stats $event_stats */
 /** @var array $projects */
 /** @var WP_User $user */
 
-/* translators: %s: Event title. */
-gp_title( sprintf( __( 'Translation Events - %s' ), esc_html( $event_title ) ) );
-gp_breadcrumb_translation_events( array( esc_html( $event_title ) ) );
-gp_tmpl_header();
-$event_page_title = $event_title;
-gp_tmpl_load( 'events-header', get_defined_vars(), __DIR__ );
+Templates::header(
+	array(
+		/* translators: %s: Event title. */
+		'html_title'  => sprintf( __( 'Translation Events - %s' ), esc_html( $event->title() ) ),
+		'page_title'  => function () use ( $event ) {
+			echo esc_html( $event->title() );
+			if ( 'draft' === $event->status() ) : ?>
+				<span class="event-label-draft"><?php echo esc_html( $event->status() ); ?></span>
+			<?php endif; ?>
+			<?php
+		},
+		'breadcrumbs' => array( esc_html( $event->title() ) ),
+		'sub_head'    => function () use ( $event, $user, $hosts ) {
+			?>
+			<span class="event-host">
+				<?php
+				if ( isset( $hosts ) ) :
+					if ( count( $hosts ) > 0 ) :
+						if ( 1 === count( $hosts ) ) :
+							esc_html_e( 'Host:', 'gp-translation-events' );
+						else :
+							esc_html_e( 'Hosts:', 'gp-translation-events' );
+						endif;
```

Comment:
We should use a plural function here and insert the list of hosts as a variable.
```suggestion
						echo esc_html( _n( 'Host: %s' , 'Hosts: %s', count( $hosts ) 'gp-translation-events' ) );

```

Reply:
After "Host:" (or "Hosts:") we display links to the profile of each host. Do you mean that we should pass that markup as the argument that will replace `%s`? I'm guessing that's not what you meant.

Reply:
Yes, see below